apiary.apib

```diff
@@ -36,44 +37,79 @@ current session status.
 
 + Response 201 (application/vnd.api+json)
 
-        {
-            "links": {
-                "self": "https://example.com/session"
-            },
-            "data": {
-                "id": "sessionid123",
-                "type": "session"
-            },
-            "related": {
-                "account": {
-                    "links": {
-                        "related": "https://example.com/session/account"
-                    },
-                    "data": {
-                        "id": "abc1234",
-                        "type": "account"
+    + Headers
+
+            Location: https://example.com/session
+
+    + Body
+
+            {
+                "links": {
+                    "self": "https://example.com/session"
+                },
+                "data": {
+                    "id": "sessionid123",
+                    "type": "session",
+                    "relationships": {
+                        "account": {
+                            "links": {
+                                "related": "https://example.com/session/account"
+                            },
+                            "data": {
+                                "id": "abc1234",
+                                "type": "account"
+                            }
+                        }
                     }
                 },
-                "profile": {
-                    "links": {
-                        "related": "https://example.com/session/account/profile"
+                "included": [{
+                    "id": "abc1234",
+                    "type": "account",
+                    "attributes": {
+                        "username": "jane-doe"
                     },
-                    "data": {
-                        "id": "profile",
-                        "type": "accountProfile"
+                    "relationships": {
+                        "profile": {
+                            "links": {
+                                "related": "https://example.com/session/account/profile"
+                            },
+                            "data": {
+                                "id": "abc1234-profile",
+                                "type": "accountProfile"
+                            }
+                        }
                     }
-                }
+                }]
             }
```

Comment:
I think the `"indcluded"` part requires `?include=account`, doesn't it? Or can we send this anyway, because in the context of sign in, it probably doesn't make sense to sign in without getting a username and an account id in response?